### DISCUSSION TITLE
有沒有可能我的藍色和你的藍色並不一樣

### DISCUSSION THREAD

**Discussion by hardtagottaname** [1101-GS4719-C/Python-Community-GS4719-C#1355] [General]
從小父母便教育我們，天空是藍色的，雲是白色的。但我一直在思考，有沒有可能我看到的藍並不是其他人眼中的藍?
科學定義的藍光是波長介於450nm到475nm的，但這也只是科學定義，有可能我看到的藍色實際上在A眼中是綠光，在B眼中卻是黃光。但到底要如何確定我看到的藍色和其他人一樣呢?
我認為，這沒有解決方法，因為顏色觀察是極為主觀的，無法從其他客觀的角度確定他的藍是否是我的藍。

**Comment by quan0715** [comment]
我認為顏色是主觀的，但色碼是絕對的請多多利用 Hex 色碼來表式任何可事話的顏色 例如 #FFFFFF #000000

**Comment by KevinCheng42** [comment]
我也覺得每個人對於顏色的觀察結果會是不同的，根據網路上的資料，因為每個人的視錐細胞對於不同顏色的敏感度不同（由攜帶不同的視蛋白所造成的），因此觀察到的顏色也有所差異～

**Comment by alicia0725** [comment]
我也覺得在每個人眼中的世界顏色都會是不一樣的，除了在大家眼中除了光線會影響，導致對每個人而言再相近的顏色都會有誤差，我也發現在大家眼中對色彩的飽和度差別很大，我眼中濃厚的色彩在他人眼中卻可能是很淺的顏色

**Comment by TW-Maverick** [comment]
我認為以後說不定可以發明一種工具，透過連結到其他人的視錐細胞或神經，讓我們以不同人的視角看世界！來貫徹「換位思考」。

老天創造的人都是獨一無二的，眼中所看到的顏色與個性都是不同的。

**Comment by shuxian12** [comment]
我覺得這種事蠻常發生的吧，因為有時候我們可能請某個人拿藍色的帽子，但他可能認為是綠色的，就會造成溝通上的誤差，也蠻常聽到幾個人聚在一起討論那個顏色到底是什麼。
另外方面也因為燈泡的燈光也會造成人對顏色有不一樣的看法

**Comment by sjh501** [comment]
我個人對於顏色就頗有障礙，眼睛沒有什麼毛病但就是會和周遭的人的顏色認知上有錯誤，最常發生的就是對於橘色和黃色每次和其他人認為的都不一樣。雖然說使用色碼表示會是最準確的，但在不同的材質上面據說也會影響顏色本身，只能說可能永遠無法知道自己和別人所認為的顏色是否相同了。

**Comment by Superlan8** [comment]
我之前在網路上常常看見一些影片，內容是在說有色盲的人戴上色盲眼鏡後的反應，我認為這件事還蠻奇妙的，他們到底戴上眼鏡後跟我們看到的世界是不是還是一樣，畢竟在那之前他們對於這些"新"顏色是陌生的，而且像作者所說，同樣都講藍色，兩個人真實所說的顏色是否又一樣，我想這種東西還真的沒有一個公正的答案。

**Comment by zephyrxin01** [comment]
我覺得很合理！每個人對於顏色的感受和理解都有所差異。就像有些人可能會認為某種顏色是天空藍，但對於其他人來說，可能會有稍微不同的認知。所以才會有色彩排列遊戲或是找不同色塊的遊戲出現！所以我蠻認同顏色是一個主觀的體驗，每個人都可能有不同的看法和感受。

**Comment by Miinteaa** [comment]
我認同色彩是主觀感受，其實每種色彩的含義也都是我們人類賦予的，不知道大家有沒有想過「藍色」為什麼會叫做「藍色」？這是不是能夠顯示在每個人的眼中，相同顏色也能夠代表不同的意義呢？其次，顏色的辨別除了先天性的敏感度以外，也能夠透過訓練來達成相近色的辨認，兩個人可能因為對於色彩的敏感度不同，對於色彩有不一樣的詮釋。

**Comment by Alexgns** [comment]
色彩實際上不太算是真實存在的，不同色彩實際上為不同波長之電磁波，並由大腦虛構出來的，因此依照不同人的大腦感知，的確使得每個人看見的色彩可能不盡相同
